shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-core-spring/shardingsphere-jdbc-core-spring-namespace/src/test/java/org/apache/shardingsphere/spring/namespace/fixture/FixtureClusterPersistRepository.java

```diff
@@ -54,8 +54,9 @@ public void persist(final String key, final String value) {
     }
     
     @Override
-    public void persistEphemeral(final String key, final String value) {
+    public boolean persistEphemeral(final String key, final String value) {
```

Comment:
It is better to return `void` value here, and catch exception in `tryLock` to return boolean value.

Reply:
OK, I will fix them.